facturacion_mexico/facturacion_fiscal/tests/test_generador_templates_fiscal.py

```
@@ -0,0 +1,428 @@
+	doc = MagicMock()
+	doc.taxes = []
+
+	def _append(table, row):
+		doc.taxes.append(dict(row))
+
+	doc.append = _append
+	doc.name = "ITT Test - _TC"
+	return doc
+
+
+def _patch_itt(mock_doc):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_patch_itt`

(ANN202)